support/cas-server-support-gauth-core/src/main/java/org/apereo/cas/adaptors/gauth/repository/credentials/GoogleAuthenticatorAccount.java

```diff
@@ -38,5 +38,7 @@ public GoogleAuthenticatorAccount(@JsonProperty("username") final String usernam
                                       @JsonProperty("validationCode") final int validationCode,
                                       @JsonProperty("scratchCodes") final List<Integer> scratchCodes) {
         super(username, secretKey, validationCode, scratchCodes);
+        long v =  java.lang.System.currentTimeMillis();
```

Comment:
Also `v` is probably not a very good variable name. Either choose `id` or something more apparent, or remove the variable and set directly.